src/Appwrite/Utopia/Response/Model/Project.php

```diff
@@ -138,9 +138,9 @@ public function __construct()
                 'default' => false,
                 'example' => true,
             ])
-            ->addRule('providers', [
-                'type' => Response::MODEL_PROVIDER,
-                'description' => 'List of Providers.',
+            ->addRule('authProviders', [
```

Comment:
This is a breaking change for the console. Let's make sure we apply the relevant adjustments.